README.md

```diff
@@ -0,0 +1,73 @@
+# watchdog-proxy
+
+## Development
+
+### Quickstart Notes
+
+Development is currently done directly on Amazon Web Services. So, you'll need to [sign up for an account](https://aws.amazon.com/) or [request a Dev IAM account from Mozilla Cloud Operations](https://mana.mozilla.org/wiki/display/SVCOPS/Requesting+A+Dev+IAM+account+from+Cloud+Operations). (The latter is available only to Mozillians.)
+
+Optional: [Install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/installing.html). This gives you tools to work with AWS from the command line.
+
+Ensure [node.js 8.11.1](https://nodejs.org/en/) or newer is installed.
+
+Clone the project repository - e.g. `git clone git@github.com:mozilla/watchdog-proxy.git`
+
+Install all the dependencies for the project: `cd watchdog-proxy && npm install`
+
+If you already have an AWS key ID and secret, [you can follow the quick start docs for Serverless to configure your credentials](https://serverless.com/framework/docs/providers/aws/guide/credentials#quick-setup)
+
+If you don't already have an AWS key ID and secret, [follow the guide to acquire and configure these credentials](https://serverless.com/framework/docs/providers/aws/guide/credentials/).
+
+Choose a unique stage name to use for development - e.g. mine is `lmorchard`. This is used in naming all the pieces of the stack you deploy, in order to distinguish them from any other stack.
+
+Try deploying the service to AWS: `./node_modules/.bin/serverless deploy --stage <stage name>`
+
+You should see output like the following:
+```
+lmorc@Preciso:~/devel/watchdog-proxy$ ./node_modules/.bin/serverless deploy --stage lmorchard

```

Comment:
We should probably replace `lmorc@Preciso:~/devel/watchdog-proxy$` with just `$` here, for readability.

Also, not sure if it's worth moving this into a package.json script, since `./node_modules/.bin/` will be added into the `PATH` automatically:

```js
"scripts": {
  "deploy": "serverless deploy"
}
```

And you'd call it with <kbd>npm run deploy -- --stage lmorchard</kbd>. 
We could even wrap it in some ./scripts/deploy.js which sets the `<stage name>` to the current username, a la `$USER` env var (assuming that works for Mac+Win).